### ISSUE TITLE
Advanced TensorBoard logging

### ISSUE THREAD

**Issue by jindrahelcl** [ufal/neuralmonkey#109]
This is somewhat an open issue, but looking at the [TensorBoard demo](https://www.tensorflow.org/tensorboard/index.html), it's obvious that our TensorBoard logging could be more informative.

Edit: since the demo is no longer available, here's a list of things to add to TB logging:
- [ ] Learning rate
- [ ] ...

**Comment by jlibovicky** [comment]
The link is broken.

**Comment by jindrahelcl** [comment]
Weird.. Let's wait and hope it'll fix itself.

**Comment by jlibovicky** [comment]
Total number of parameters could je also an interesting statistic.

**Comment by jlibovicky** [comment]
It's not possible to add string comment to experiments in TensorBoard, is it?

**Comment by jindrahelcl** [comment]
A simple thing you can do is to assign the experiment its name when calling the tensorboard executable, as in:

`tensorboard --logdir kocka:experiments/exp1,pejsek:experiments/exp2`

I don't know about other ways.

**Comment by jlibovicky** [comment]
So, how can we add information like learning rate or number of parameters into TB?

**Comment by jindrahelcl** [comment]
Since learning rate may not be constant, we could plot as another summary. I'd rather not plot number of parameters as a constant function, though.. :-)

In this issue, I was thinking more the actual values of the parameters, or histograms - we plot histograms of their gradients, why not just plot the histograms of the parameters themselves?..
